### ISSUE TITLE
Make formatting number optional

### ISSUE THREAD

**Issue by stephenmoloney** [mvalitov/phone_number#1]
Hi,

On this [line](https://github.com/mvalitov/phone_number/blob/master/lib/phone_number.ex#L20), 
it seems alphabet characters are removed.

Is this something the user would or could expect - perhaps some of the formatting could be made optional?

BTW, I'm not sure if this is a good idea or not. Just looking for your thoughts/

**Comment by mvalitov** [comment]
indeed, this is an interesting thought. But while it is not clear, whether it is necessary to change it, therefore I think it is necessary to wait while someone will inform on such problem
